pkg/extensions/search/common/common_test.go

```diff
@@ -1059,6 +1059,135 @@ func TestGetReferrersGQL(t *testing.T) {
 
 		So(referrersResp.ReferrersResult.Referrers[0].Digest, ShouldEqual, artifactManifestDigest)
 	})
+
+	Convey("referrers for image index", t, func() {
+		port := GetFreePort()
+		baseURL := GetBaseURL(port)
+		conf := config.New()
+		conf.HTTP.Port = port
+		conf.Storage.RootDirectory = t.TempDir()
+		conf.Storage.GC = false
+
+		defaultVal := true
+		conf.Extensions = &extconf.ExtensionConfig{
+			Search: &extconf.SearchConfig{BaseConfig: extconf.BaseConfig{Enable: &defaultVal}},
+			Lint: &extconf.LintConfig{
+				BaseConfig: extconf.BaseConfig{
+					Enable: &defaultVal,
+				},
+			},
+		}
+
+		gqlEndpoint := fmt.Sprintf("%s%s?query=", baseURL, graphqlQueryPrefix)
+
+		conf.Extensions.Search.CVE = nil
+
+		ctlr := api.NewController(conf)
+		ctlrManager := NewControllerManager(ctlr)
+		ctlrManager.StartAndWait(port)
+		defer ctlrManager.StopServer()
+
+		// =======================
+
+		multiarch, err := GetRandomMultiarchImage("multiarch")
+		So(err, ShouldBeNil)
+		repo := "artifact-ref"
+
+		err = UploadMultiarchImage(multiarch, baseURL, repo)
+		So(err, ShouldBeNil)
+
+		indexBlob, err := json.Marshal(multiarch.Index)
+		So(err, ShouldBeNil)
+		indexDigest := godigest.FromBytes(indexBlob)
+		indexSize := int64(len(indexBlob))
+
+		subjectDescriptor := &ispec.Descriptor{
+			MediaType: "application/vnd.oci.image.manifest.v1+json",
```

Comment:
The digest belongs to an index, but MediaType is a manifest?
Otherwise PR looks good.